extensions/amp-twitter/0.1/amp-twitter.md

```diff
@@ -0,0 +1,131 @@
+---
+$category@: social
+formats:
+  - websites
+  - stories
+teaser:
+  text: Displays a Twitter Tweet or Moment.
+---
+
+# amp-twitter
+
+## Behavior
+
+The `amp-twitter` component allows you to embed a Tweet or Moment for the specified Twitter ID.
```

Comment:
..the specified Tweet ID?

Reply:
Looks like this is a pre-existing issue, I'll fix in another PR.